defaults/validate_test.go

```diff
@@ -0,0 +1,84 @@
+package defaults
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidate(t *testing.T) {
+	type serverConfiguration struct {
+		URL           string `default:"http://localhost:8065" validate:"url"`
+		Email         string `default:"sysadmin@sample.mattermost.com" validate:"email"`
+		AdminPassword string `default:"Sys@dmin-sample1" validate:"text"`
```

Comment:
What does validate:text do ?

Reply:
It's just checks if the string is empty, I'm open to naming suggestions :)

Reply:
How about "notempty" ?

Reply:
Much better tbh 👍